core/lib/dal/src/consensus_dal.rs

```diff
@@ -317,7 +323,39 @@ impl ConsensusDal<'_, '_> {
         else {
             return Ok(None);
         };
-        Ok(Some(zksync_protobuf::serde::deserialize(row.certificate)?))
+        Ok(Some(zksync_protobuf::serde::deserialize(
+            row.certificate.unwrap(),
```

Comment:
with non null removed, you shouldn't just unwrap

Reply:
Fixed here: https://github.com/matter-labs/zksync-era/pull/2518/commits/65d3fa3d928d1667e3e4b0bd466d7d4c392dfaa3